kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1231,13 +1231,20 @@ object KyuubiConf {
       .checkValue(_ > 0, "must be positive number")
       .createWithDefault(Duration.ofMinutes(5).toMillis)
 
-  val KUBERNETES_SPARK_DELETE_DRIVER_POD_ON_TERMINATION_ENABLED: ConfigEntry[Boolean] =
-    buildConf("kyuubi.kubernetes.spark.deleteDriverPodOnTermination.enabled")
-      .doc("If set to true then Kyuubi server will delete the spark driver pod after " +
-        s"the application terminates for ${KUBERNETES_TERMINATED_APPLICATION_RETAIN_PERIOD.key}.")
+  val KUBERNETES_SPARK_DELETE_DRIVER_POD_ON_TERMINATION: ConfigEntry[String] =
+    buildConf("kyuubi.kubernetes.spark.deleteDriverPodOnTermination")
```

Comment:
let's rename the configuration name to `kyuubi.kubernetes.spark.cleanupTerminatedDriverPod`

Reply:
OK